docs/quant/3_msstats_workflow_2_run.md

```diff
@@ -0,0 +1,154 @@
+
+Once your files are fully uploaded to your user account directory at MassIVE or you know which MSV or RMSV has quantification file you want to reanalyze, you are ready to run **MSstats workflow** in MassIVE.quant.
+
+### 2.1 Workflow Selection
+
+First, go to the main webpage for [MassIVE](https://massive.ucsd.edu/ProteoSAFe/static/massive.jsp). Click **Submit your data** in the `Submit Data` section. 
+
+<center>
+![](img/run_msstats_workflow/msstats_workflow_step0.png)
+</center>
+
+It brings you to the workflow input form. Select **MSstats (1.3)** from the drop-down button in the `Workflow`. Then the webpage will be changed as below. You should add the title in the `Title` (at least 30 characters).
```

Comment:
**MSstats (1.3)** ->  **MSstats (1.3)** (1.3 refers to the version at the time of writing, please use whatever version is in the dropdown, relevant sections of this documentation will be updated as the version is)

Reply:
Or really anything along those lines, we just don't want to get an email when we make a minor bug fix and the version is 1.3.1